[FIX] github_connector: Add credentials to clone a private repository. (#58)
OCA/interface-github: github_connector/readme/USAGE.rst

   .. image:: ../static/description/github_repository_branch_list.png

Download a private repository
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

If you want to fetch the source code from a private repository, you need a token (instead of login/password) with the right scopes:

#. Go to 'Github Account' / 'Settings' / 'Developer settings' / 'Personal access tokens' / 'Generate new token'.

>    .. image:: https://raw.githubusercontent.com/OCA/interface-github/12.0/github_connector/static/description/generate_new_token.png

Reviewer: Use this to avoid having to update the link manually on each version bump:

```suggestion
   .. image:: ../static/description/generate_new_token.png
```

The bot will convert that to an absolute path when building the README.

#. In the 'New personal access token' you have to select 'repo scope'.

>    .. image:: https://raw.githubusercontent.com/OCA/interface-github/12.0/github_connector/static/description/personal_access_token.png

Reviewer: ```suggestion
   .. image:: ../static/description/personal_access_token.png
```

> Scopes:
> 
> * **(no scope)** : Grants read-only access to public information (includes public user profile info, public repository info, and gists).
> * **repo** : Grants full access to private and public repositories.
> * **repo:status** : Grants read/write access to public and private repository commit statuses.
> * **repo_deployment** : Grants access to deployment statuses for public and private repositories.
> * **public_repo** : Limits access to public repositories. That includes read/write access to code, commit statuses.
> * **repo:invite** : Grants accept/decline abilities for invitations to collaborate on a repository.

Reviewer: ```suggestion
Add one of these scopes to your token:

* **(no scope)** : Grants read-only access to public information (includes public user profile info, public repository info, and gists).
* **repo** : Grants full access to private and public repositories. `See here why is this needed <https://youtu.be/07F2pHfvHvA>`__.
```
